common/views/components/ButtonSolid/ButtonSolid.tsx

```diff
@@ -154,8 +158,20 @@ export const SolidButton = styled(BaseButton).attrs<SolidButtonProps>(
         props?.colors?.border || props.theme.buttonColors.default.border
       )};
 
+  content: '${props => props.size}';
+
+  ${props =>
+    props.size === 'small' &&
+    `
+    padding: 8px 12px;
+  `}
+  ${props =>
+    props.size === 'medium' &&
+    `
+    padding: 13px 20px;
+  `}
   ${props =>
-    props.isBig &&
+    props.size === 'large' &&
     `
     padding: 14px;
```

Comment:
Wouldn't this be smaller than a medium button (all around)?

Reply:
In terms of volume, I guess it would, yes – just two pixels taller. I think I mentioned in the frontend meeting that there's only one (maybe two) instances of an `isBig` button in any event, and it should probably cease to exist. But this was mostly to achieve parity with what's there currently.

Reply:
Cool, I'll just add it to my list of "this is possible but should it be?" 😄